apps/web/src/components/dashboard/container.tsx

```diff
@@ -9,21 +9,34 @@ import {
 import { cn } from "@/lib/utils";
 import { Skeleton } from "../ui/skeleton";
 
-interface CardProps extends React.HTMLAttributes<HTMLDivElement> {
-  title: string;
+interface CardProps
+  extends Omit<React.HTMLAttributes<HTMLDivElement>, "title"> {
+  title: React.ReactNode;
   description?: string;
+  actions?: React.ReactNode | React.ReactNode[];
```

Comment:
Better for visibility and layout instead of using an `absolute` position on the `action-button`s.